imap_processing/lo/l1c/lo_l1c.py

```diff
@@ -68,15 +68,12 @@ def lo_l1c(sci_dependencies: dict, anc_dependencies: list) -> list[xr.Dataset]:
             full_counts, l1b_goodtimes_only
         )
     pset.attrs = attr_mgr.get_global_attributes(logical_source)
-    # TODO: Temp fix before adding attribute variables.
-    #  CDF won't open if DEPEND_0 is not deleted currently.
-    del pset["epoch"].attrs["DEPEND_0"]
 
     pset = pset.assign_coords(
         {
-            "energy": np.arange(1, 8),
-            "longitude": np.arange(3600),
-            "latitude": np.arange(40),
+            "esa_energy_step": np.arange(1, 8),
+            "spin_angle": np.arange(3600) / 3600 + 0.05,
```

Comment:
These don't look right. I prefer the way you implemented it above...
```suggestion
            "spin_angle": np.arange(0 + bin_spacing_deg / 2, 360, bin_spacing_deg),
            "off_angle": np.arange(-2 + bin_spacing_deg / 2, 2, bin_spacing_deg),
```

Or from your suggestion on Hi code, should you be using np.linspace. I forget the exact reason that you made that suggestion.

Reply:
linspace is supposed to be more stable for floating point numbers according to the docs. I have upated this with globals to correspond with the Hi version.